src/components/FilterSingleSelect/FilterSingleSelect.stories.tsx

```diff
@@ -14,6 +16,9 @@ const options = [
 ]
 
 export const example = (): ReactElement => {
+
+	const required = boolean("Required", false);

```

Comment:
Please add a knob for changing the value / default value, because then whoever is looking at the storybook will never be able to see how the required logic works, they will only see the error message (unless they select an option and then click on the required knob). I'll also recommend adding a kitchen sink story with both types of filter, a required and a non required.